README.md

```diff
@@ -4,63 +4,65 @@
 
 Our goal is to make it easy for XNA developers to create cross-platform games with extremely high code reuse.
 
-## Supported Platforms
+### Links
 
-* iOS (including Retina displays)
-* Android
-* Windows (OpenGL & DirectX)
-* Mac OS X
-* Linux
-* [Windows Store Apps](https://dev.windows.com/) (for Windows 8 and Windows RT)
-* [Windows Phone 8](http://dev.windowsphone.com)
-* [Windows Phone 8.1](http://dev.windows.com)
-* [Windows 10 UWP](http://dev.windows.com)
-* [OUYA](https://www.ouya.tv/), an Android-based gaming console
+ * Forum can be found at: [community.monogame.net](http://community.monogame.net/)
+ * Chat with the community at:&nbsp; [![Join the chat at https://gitter.im/mono/MonoGame](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mono/MonoGame?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+ * Documentation is available at: [www.monogame.net/documentation/](http://www.monogame.net/documentation/)
+ * [Download Binaries](http://www.monogame.net/downloads/).
+ * Follow [@MonoGameTeam on Twitter](https://twitter.com/monogameteam).
 
-## Quick Start
+### Supported Platforms
 
-There are a few pre-requisites that you should be aware of for the various platforms. Please check them over [MonoGame](http://www.monogame.net/downloads/) website.
+Desktop:
+ * Windows Store Apps
+ * Windows (OpenGL & DirectX)
+ * Linux
+ * Mac OS X
 
-There are four quick start options available:
-* Download the current stable release from [our website](http://monogame.net/downloads).
-* Download the latest "unstable" installer from [our build server](http://teamcity.monogame.net/viewLog.html?buildTypeId=MonoGame_DevelopWin&buildId=lastSuccessful&tab=artifacts&buildBranch=%3Cdefault%3E&guest=1).
-* Download the latest [source code](https://github.com/mono/MonoGame/archive/develop.zip).
-* Fork and clone the repo: `https://github.com/mono/MonoGame.git`.
+Mobile:
+ * Android
+ * iOS
+ * Windows Phone
 
-### Solutions & Projects
+Console:
+ * Playstation 4
```

Comment:
It should be "PlayStation 4" also we can add "PlayStation Vita" here too.

For those platforms we need to add a note like "contact your Sony account manager for access".